atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/Transaction.java

```diff
@@ -130,6 +130,21 @@
     Map<byte[], Iterator<Map.Entry<Cell, byte[]>>> getRowsColumnRangeIterator(
             TableReference tableRef, Iterable<byte[]> rows, BatchColumnRangeSelection columnRangeSelection);
 
+    /**
+     * Returns an iterator over cell-value pairs within {@code tableRef} for the specified {@code rows}, where the
+     * columns fall within the provided {@link BatchColumnRangeSelection}. The single provided {@link BatchColumnRangeSelection}
+     * applies to all of the rows.
+     *
+     * The returned iterator is guaranteed contains cells sorted first in order of column, then in order of row.
```

Comment:
Probably worth clarifying that order of row here is the lexicographic ordering of rows (which is crucially different from that in e.g. `getRowsColumnRangeIterator`).

Reply:
The inconsistency with the other endpoints is a little uncomfortable, so it might be better to adhere to that unless it's too costly to do so.